tensorboard/BUILD

```diff
@@ -360,6 +368,18 @@ py_library(
     ],
 )
 
+py_test(
+    name = "proto_test",
+    size = "small",
+    srcs = ["proto_test.py"],
+    srcs_version = "PY2AND3",
```

Comment:
Let's add below this line a manual tag so this doesn't run against tf-nightly on travis, where it will probably break more often than useful - we can explicitly run the test as part of the release cycle instead.

```
tags = [
    "manual",
],
```